### PR TITLE
Ana/use ModalDeposit for Polkadot seconding

### LINKED ISSUE #4899: Create a new ModalVote for Polkadot democracy proposals (seconding)
Since #4566 is already huge, the following enhancements will be treated in separate PRs:

We could also leave it as it is. But maybe not that maintainable, I agree. Although the `v-ifs` are what I find most confusing, the rest is OK.

Comment: https://github.com/luniehq/lunie/pull/4566#discussion_r487289116

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #4899 

**Description:**

Lots of work to do (validation, warning message...). Also, the original ModalVote for Polkadot is gone, strangely :shrug: Need to look further into that

Go away, old vote modal!

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by jbibla** [comment]
how is this different than depositing?

**Comment by Bitcoinera** [comment]
OK, this is how it is looking now, simply using the ModalDeposit:

image

**Comment by faboweb** [comment]
🚀 Deployed on https://5f634b449f65b09ab5241db2--lunieio.netlify.app

File: app/src/ActionModal/components/ModalBackPolkadotProposal.vue [hunk 1]
```diff
@@ -0,0 +1,171 @@
+<template>
```
Comment (jbibla) [inline app/src/ActionModal/components/ModalBackPolkadotProposal.vue:1]: delete this component

File: app/src/ActionModal/components/ModalDeposit.vue [hunk 2]
```diff
@@ -25,6 +25,7 @@
         id="amount"
         v-model="amount"
```
Comment (jbibla) [inline app/src/ActionModal/components/ModalDeposit.vue:26]: i think we can remove the disabling logic. 

if polkadot: amount = 100

File: app/src/ActionModal/components/ModalDeposit.vue [hunk 3]
```diff
@@ -25,6 +25,7 @@
...
         v-focus
+        :disabled="numberOfSeconds !== 0"
```
Comment (jbibla) [inline app/src/ActionModal/components/ModalDeposit.vue:28]: disable for polkadot

File: app/src/ActionModal/components/ModalDeposit.vue [hunk 4]
```diff
@@ -150,6 +156,9 @@ export default {
   },
   methods: {
     open() {
+      if (this.numberOfSeconds !== 0) {
```
Comment (jbibla) [inline app/src/ActionModal/components/ModalDeposit.vue:159]:
```suggestion
      if (polkadot) {
```

File: app/src/components/governance/PageProposal.vue [hunk 5]
```diff
@@ -30,20 +30,27 @@
...
         :proposal-title="proposal.title || ''"
-        :denom="parameters.depositDenom"
+        :denom="parameters.depositDenom || currentNetwork.stakingDenom"
+        :number-of-seconds="
+          isPolkadotDemocracy ? Number(proposal.detailedVotes.votesSum) : 0
+        "
         @success="() => afterDeposit()"
       />
-      <ModalVoteSubstrate
+      <ModalVotePolkadot
```
Comment (jbibla) [inline app/src/components/governance/PageProposal.vue:47]: why switch from substrate to polkadot?

Comment (Bitcoinera) [reply app/src/components/governance/PageProposal.vue:47]: Discussing in another PR with Fabo about this, we agreed that in the end it is more simple to just talk about `Polkadot` and forget about `Substrate`

File: app/src/ActionModal/components/ModalDeposit.vue (hunk 3 above)
Comment (Bitcoinera) [reply app/src/ActionModal/components/ModalDeposit.vue:28]: OK, going to change it

File: app/src/ActionModal/components/ModalBackPolkadotProposal.vue (hunk 1 above)
Comment (Bitcoinera) [reply app/src/ActionModal/components/ModalBackPolkadotProposal.vue:1]: true

**Review by jbibla — APPROVED** [review]

**Review by jbibla — APPROVED** [review]

**Pull request merged**